### DISCUSSION TITLE
CSD in Xfce

### DISCUSSION THREAD

**Discussion by Flashwalker** [lah7/gtk3-classic#92] [General Chat]
Is there anyone who uses Xfce (4.18)?
Do you have CSD in Xfce windows (e.g.: xfce4-settings-manager)?
![image](https://user-images.githubusercontent.com/2188215/222867970-29ede2ed-3b6f-4a28-9d56-076e82d2c654.png)

**Comment by ghost** [comment]
I use some XFCE apps/style to call my desktop environment XFCE (Wayfire as a backing wayland compositor). However, I also use NixOS to manage all of my ephemeral settings (so I don't use the XFCE settings manager).

Compiling this app from my distro's upstream repository (v4.18.1) I don't see the spacing your screenshot shows (above the GtkSearchEntry). Is this perhaps added by your theme? Additionally, is this specific to gtk3-classic?

**Reply by Flashwalker** [reply]
> Is this perhaps added by your theme?

No, i tried different themes

> is this specific to gtk3-classic?

Need testing

**Reply by Flashwalker** [reply]
> is this specific to gtk3-classic?

I tested it. Just purged gtk3-classic ppa and packages:
```
gir1.2-gtk-3.0 gtk-update-icon-cache libgail-3-0 libgtk-3-0 libgtk-3-bin libgtk-3-common libgtk-3-dev
```


That CSD panel in Xfce windows has disappeared because it now uses (or does not use) its own window border. So, now CSD substitutes the window manager's window border
![image](https://user-images.githubusercontent.com/2188215/225221119-0923238e-00dd-461f-8103-3d63ffd91c4f.png)
So i think it is actually a **gtk3-classic** related issue. Despite i `unset GTK_CSD` in `~/.profile`.
